controllers/storagecluster/exporter.go

```diff
@@ -142,14 +148,16 @@ func getMetricsExporterServiceMonitor(instance *ocsv1.StorageCluster) *monitorin
 			},
 			Endpoints: []monitoringv1.Endpoint{
 				{
-					Port:     portMetrics,
-					Path:     metricsPath,
-					Interval: scrapeInterval,
+					Port:           portMetrics,
+					Path:           metricsPath,
+					Interval:       scrapeInterval,
+					RelabelConfigs: []*monitoringv1.RelabelConfig{&relabelConfig},
 				},
 				{
-					Port:     portExporter,
-					Path:     metricsPath,
-					Interval: scrapeInterval,
+					Port:           portExporter,
+					Path:           metricsPath,
+					Interval:       scrapeInterval,
+					RelabelConfigs: []*monitoringv1.RelabelConfig{&relabelConfig},
```

Comment:
We don't need exporter metrics to be standarized.
Let's remove this. It might save some relabling cycles.

Reply:
Okay. Sure.